plugins/woocommerce-admin/client/products/sections/test/utils.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Internal dependencies
+ */
+import { isValidSlug } from '../utils';
+
+describe( 'product section utils', () => {
+	describe( 'isValidSlug', () => {
```

Comment:
As mentioned above, I think that we should respect the current validation constraints in core.